sample2/counter/api.yaml

```diff
@@ -0,0 +1,108 @@
+---
+tableStorage:
+    table: apicountertablelocal
+    partitionKey: items
+    connection: AzureWebJobsStorage
+
+language: CSharp
+
+commonCode: |
+        #r "Microsoft.WindowsAzure.Storage"
+        using System.Net;
+        using Microsoft.WindowsAzure.Storage.Table;
+        using System.Net.Http;
+        public class Counter : TableEntity
+        {
+            public int Value { get; set; }
+        }
+    
+functions:
+    - name: initialize
+      trigger: "PUT@counteryaml/{countername}"
+      bindings:
+        - "counter:table-out"
+      codeLocation: "initialize.csx"
+    - name: get
+      trigger: "GET@counteryaml/{countername}"
+      bindings:
+        - "counters:table-in"
+      code: |
+        public static async Task<HttpResponseMessage> Run(HttpRequestMessage req, TraceWriter log, IQueryable<Counter> counters, string countername)
+        {
+            HttpResponseMessage res = null;
+            try
+            {
+                Counter counter = counters.Where(c => c.RowKey.Equals(countername)).FirstOrDefault();
+                if(counter != null)
+                {
+                    res = new HttpResponseMessage(HttpStatusCode.OK)
```

Comment:
You can move res = new HttpResponseMessage(HttpStatusCode.OK) out of the IF statement since it is repeat in the else statement.